docs/pages/access-controls/access-request-plugins/notification-routing-rules.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Routing Access Request notifications
+description: How to set up Teleport's Access Monitoring Rules to route Access Request notifications
+---
+
+With Teleport's Access Monitoring Rules Access Request notifications
+can be routed to plugins based on based on several conditions.
+For example, if you wish to notify different recipients e.g Slack channels,
+based on what roles or resources are being accessed, or based on what user
+is making the request.
+
+Currently only the Slack access plugin is supported.
```

Comment:
What's the difference between this file and `access-monitoring-rules.mdx`? There seems to be a ton of duplication.

Reply:
This should be the main file, accidentally recommitted access-monitoring-rules.mdx after a rename. 
Have moved everything into notification-routing-rules and remove the duplication